src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/package.json

```diff
@@ -8,7 +8,8 @@
     "build:ssr": "ng run Company.WebApplication1:server:dev",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "check": "npm run lint"
```

Comment:
I'm not certain why we're adding this `check` script instead of directly running `npm run lint` elsewhere. This is not an objection - I'm fine with it, just saying I'm not sure what the design intent is.